test/common/upstream/health_check_fuzz.proto

```diff
@@ -9,8 +9,7 @@ import "google/protobuf/empty.proto";
 
 message HttpRespond { //TODO: Split this across headers, body, and trailers?
   test.fuzz.Headers headers = 1;
-  //https://github.com/nodejs/http-parser/blob/master/http_parser.c#L852
-  string status = 2 [(validate.rules).string = {pattern: "[0-9]+$*", min_len: 1}];
+  uint64 status = 2;
```

Comment:
If you need to preserve the non-empty-ness of this, gt would suffice

Reply:
Decided not to, as it defaults to zero. Would it be better if I added?

Reply:
Ah, gotcha! Thanks, I think it's fine as is then.